index.js

```diff
@@ -1,43 +1,62 @@
 /* eslint-env node */
 'use strict';
+const Funnel = require('broccoli-funnel');
+const getDebugMacros = require('./src/debug-macros').debugMacros;
 
-function debugMacros(features) {
-  let plugins = [
-    [
-      require.resolve('babel-plugin-debug-macros'),
-      {
-        flags: [
-          {
-            source: 'ember-m3/feature-flags',
-            flags: features,
-          },
-        ],
-      },
-      '@ember-data/canary-features-stripping',
-    ],
-  ];
-
-  return plugins;
+/*
+ shouldIncludeChildAddon runs during
+ addon init so does not have access to
+ the host application instance.
+*/
+function hasDataPackage(addon) {
+  let addons = addon.project.addonPackages;
+  return addons['ember-data'] !== undefined;
 }
 
 module.exports = {
   name: 'ember-m3',
```

Comment:
Although it hasn't been done so far, it may be worth adding a guard erroring if more then 1 ember-m3 is installed. Enforcing the highlander stuff. Otherwise, if two are installed, and they each bring their own ember-data stuff with, sadness will occur.

Reply:
I'll add that in a follow up 👍